index.html

```diff
@@ -8,50 +8,50 @@
     <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    <top_nav>
-        <item_nav>
-            <frame4>
-                <portfolio>portfolio</portfolio>
-            </frame4>
-            <frame3>
-                <about>About</about>
-            </frame3>
-            <frame5>
-                <contact>Contact</contact>
-            </frame5>
+    <to1_nav1 id="to1_nav1">
+        <item_nav id="item_nav">
+            <fram4>
+                <portf id="portf">portfolio</portf>
+            </fram4>
+            <fram3>
+                <about id="about">About</about>
+            </fram3>
+            <fram5>
+                <contact id="contact">Contact</contact>
+            </fram5>
         </item_nav>
 
-        <name>
-            <johndoe>John doe</johndoe>
-        </name>
+        <nam id="nam">
+            <johndoe id="johndoe">John doe</johndoe>
+        </nam>
```

Comment:
- [ ] Kindly I recommend you to use `a` tag instead of `nam` tag for logo , This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Reply:
changed nam to a as requested